JDBC/src/queryPackage/QueryClass.java

```diff
@@ -0,0 +1,142 @@
+package queryPackage;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import pOJOClasses.Title;
+import connection.ConnectionUtil;
+
+public class QueryClass {
+
+	private static Connection con = null;
+	private static PreparedStatement ps = null;
+	private static ResultSet rs = null;
+	private static PreparedStatement psUpdate = null;
+
+	public static void initConnection() {
+		ConnectionUtil conUtil = new ConnectionUtil();
+		con = conUtil.getConnection();
+	}
+
+	public static void setAutocommit(boolean autoCommit) {
+
+		try {
+			con.setAutoCommit(autoCommit);
+		} catch (SQLException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
+
+	public static void rollback() {
+
+		try {
+			con.rollback();
+		} catch (SQLException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+
+	}
+
+	public static void closeConnection() {
+
+		if (con != null)
+			try {
+				con.close();
+			} catch (SQLException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+		if (ps != null) {
+			try {
+				ps.close();
+			} catch (SQLException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+
+		}
+		if (psUpdate != null) {
+			try {
+				psUpdate.close();
+			} catch (SQLException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
+			}
+
+		}
+
+	}
+
+	public static ArrayList<Title> getListOfBookByAuthorName(String author_name) {
+		ArrayList<Title> listOfTitles = new ArrayList<Title>();
+		String query = "select t.* from title t,title_author ta, author a where t.title_id=ta.title_id and ta.author_id=a.author_id and a.author_name=?";
+
+		try {
+			ps = con.prepareStatement(query);
+			ps.setString(1, author_name);
+			System.out.println(ps.execute());
+			rs = ps.executeQuery();
+
+			while (rs.next()) {
```

Comment:
ACTION: The pojos should be instantiated in the util class only.
The basic thing which needs to be understood as per the assignment guidelines is that the API should be so robust that when some one wants any records from the DB, he should simply invoke the API by passing the query and retrieving the result. The intermediate process should be done in the util class.